modules/contact-form/grunion-contact-form.php

```diff
@@ -1674,7 +1611,7 @@ function __construct( $attributes, $content = null, $form = null ) {
 			if ( empty( $id ) ) {
 				$id = sanitize_title_with_dashes( 'g' . $form_id . '-' . $unescaped_label );
 				$i = 0;
-				$max_tries = 24;
+				$max_tries = 99;
```

Comment:
Yeah, this whole thing looks botched, tests are failing, will fix it. Thanks for taking a look!